app/server.py

```diff
@@ -63,7 +63,8 @@ def index():
     return flask.render_template(
         'index.html',
         form=forms.FoodPreferenceForm(),
-        preferences=preferences)
+        preferences=preferences,
+        upcoming=[details[2].lower() for details in utils.get_upcoming_meals_for_user(current_user)])
```

Comment:
Can we use a `namedtuple` instead of referencing `[2]` since in context I have no idea what that means and why we're ignoring the first two values